srcopsmetrics/cli.py

```diff
@@ -25,18 +25,28 @@
 from srcopsmetrics.bot_knowledge import visualize_project_results
 from srcopsmetrics.evaluate_scores import ReviewerAssigner
 
+from github import Github
 
 _LOGGER = logging.getLogger("aicoe-src-ops-metrics")
 logging.basicConfig(level=logging.INFO)
 
+_GITHUB_ACCESS_TOKEN = os.getenv("GITHUB_ACCESS_TOKEN")
```

Comment:
Isn't it already present in `GitHubKnowledge` class?

Reply:
I've added get_repositories function to GitHubKnowledge. Does it make sense now?

Reply:
Yes great!